### ISSUE TITLE
No Red Connection when taking off the Potentiometers

### ISSUE THREAD

**Issue by sethenstien** [ramapcsx2/gbs-control#131]
So I took off the Potentiometers just as shown and now I can't get a Red Signal. Any idea what happened. 
![image0](https://user-images.githubusercontent.com/56380280/87979179-441f5d00-ca97-11ea-8d1c-0832befeb190.jpg)
![image2](https://user-images.githubusercontent.com/56380280/87979186-484b7a80-ca97-11ea-9b1e-74ab7a296dfb.jpg)
![image3](https://user-images.githubusercontent.com/56380280/87979190-4a153e00-ca97-11ea-8a9e-685d7a707773.jpg)

**Comment by rma945** [comment]
I have the same issue, after removing the potentiometers I have lost the red color. But now, I have also lost a green color too..

**Comment by ramapcsx2** [comment]
Most likely you've shorted a color to ground by accident. Use a multimeter beep test to find out.

**Comment by rma945** [comment]
Yes, you are right, Unfortunately, the battery in my multimeter has died, so I can't check shorts at the board after soldering bridges. But now I have tested the board, found shorts and re-solder bridges, so everything is good now. btw, Rama, thank you for that awesome firmware!
